src/wp-includes/blocks.php

```diff
@@ -60,10 +64,11 @@ function generate_block_asset_handle( $block_name, $field_name, $index = 0 ) {
 
 	$field_mappings = array(
 		'editorScript' => 'editor-script',
-		'script'       => 'script',
-		'viewScript'   => 'view-script',
 		'editorStyle'  => 'editor-style',
+		'script'       => 'script',
 		'style'        => 'style',
+		'viewScript'   => 'view-script',
+		'viewScriptModule' => 'view-script-module',
```

Comment:
It's interesting that CI didn't complain about aligning arrow keys in the array as before.

Reply:
I guess there were warnings, but I'm also surprised. I ran phpcbf and fixed all the alignment issues in this file.